dagshub/auth/tokens.py

```diff
@@ -281,6 +303,24 @@ def _store_cache_file(self):
             logger.error(f"Error while storing DagsHub token cache: {traceback.format_exc()}")
             raise
 
+    def _print_accessing_as(self, user):
+        """
+        This function prints a message to the log that we are accessing as a certain user.
+        It does this only once per command, to avoid spamming the logs.
+        It called after successful token validation.
+
+        """
+        if self._accessing_as_was_printed:
+            return
+
+        username = user.get("username")
+        if username is None:
+            # todo handle error ? it should be defined
+            return
```

Comment:
Something probably went SUPER wrong if we got the response from the API, but the structure of the dictionary is different

I say just skip the whole check and just do a regular dict access. If such a weird case happens, it's probably a backend error and there's no point in trying to handle it in the client
```suggestion
        username = user["username"]
```